invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""MARCXML based Schema for Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import bleach
+from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from marshmallow import Schema, fields, missing
+
+from ..ui.schema import current_default_locale
+from ..utils import get_vocabulary_props
+
+
+class MARCXMLSchema(Schema):
+    """Schema for records in MARC."""
+
+    contributors = fields.Method("get_contributors")
+    titles = fields.Method("get_titles")
+    creators = fields.Method("get_creators")
+    identifiers = fields.Method("get_identifiers")
+    relations = fields.Method("get_relations")
+    rights = fields.Method("get_rights")
+    dates = fields.Method("get_dates")
+    subjects = fields.Method("get_subjects")
+    descriptions = fields.Method("get_descriptions")
+    publishers = fields.Method("get_publishers")
+    types = fields.Method("get_types")
+    sources = fields.Method("get_sources")
+    coverage = fields.Method("get_locations")
```

Comment:
I think that's not listed https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/schemas/metadata.py#L330 . Can you also that we don't miss any other fields?

Reply:
So I've done a sort of splitting of some fields for clarity / reusing existing schema names
"publisher" is present on the schema (hence publishers is just a list of them)
"types" is present as "resource_type" on the metadata schema
"sources" is present as "references"  on the metadata schema
"coverage" is present as "locations"  on the metadata schema

should these variables be renamed to their corresponding metadata schema counter-part or should i leave them as is (IE like in the other schema.py files)?

Also i have added the missing fields from the metadata schema

Reply:
I think is good, if you want for clarity you can put it in the docstring of the function. For example, "locations corresponds to coverage in the standard".